app/scripts/migrations/078.test.js

```diff
@@ -0,0 +1,48 @@
+import { TEST_CHAINS } from '../../../shared/constants/network';
+import migration78 from './078';
+
+describe('migration #78', () => {
```

Comment:
We typically like to add some tests here to ensure that nothing changes unexpectedly, like in the case where advancedGasFee isn't set for some reason, or make sure unaffected keys do not get changed in anyway. Please add a few more test cases even they feel redundant, use the other tests for examples

Reply:
Got it!
Added here: 8677314c1999ffa3995c3cb693e64a8d9841009d